common/djangoapps/mailchimp_pipeline/tests/helpers.py

```diff
@@ -38,12 +41,13 @@ def create_organization_partner_object(user):
     partner.save()
     return partner
 
+
 def generate_mailchimp_url(root_url, email):
     """
-        Convert email into encrypted email hash
-        :param root_url: MailChimp root URL
-        :param list_id: MailChimp users list_id
-        :return: hashed email
+    Convert email into encrypted email hash
+    :param root_url: MailChimp root URL
+    :param email: MailChimp user email
+    :return: formatted path with url and email hash
```

Comment:
Lets use the same format for docstring here as well as mentioned above

Reply:
The return type and the arguments description of the docstring was somewhat misleading. I made this change to avoid this misinformation.